File: base/src/main/java/me/shetj/base/http/callback/EasyProgressCallBack.java
package me.shetj.base.http.callback;

import android.support.annotation.Keep;

import com.zhouyou.http.callback.ProgressDialogCallBack;
import com.zhouyou.http.subsciber.IProgressDialog;

@Keep
public class EasyProgressCallBack<T> extends ProgressDialogCallBack<T> {

	public EasyProgressCallBack(IProgressDialog progressDialog) {
		super(progressDialog);
	}

	public EasyProgressCallBack(IProgressDialog progressDialog, boolean isShowProgress, boolean isCancel) {
		super(progressDialog, isShowProgress, isCancel);
	}

	@Override
	public void onSuccess(T t) {

	}
}

File: base/src/main/java/me/shetj/base/http/callback/EasyProgressCallBack.kt
package me.shetj.base.http.callback

import androidx.annotation.Keep

import com.zhouyou.http.callback.ProgressDialogCallBack
import com.zhouyou.http.subsciber.IProgressDialog

@Keep
class EasyProgressCallBack<T> : ProgressDialogCallBack<T> {

    constructor(progressDialog: IProgressDialog) : super(progressDialog) {}

    constructor(progressDialog: IProgressDialog, isShowProgress: Boolean, isCancel: Boolean) : super(progressDialog, isShowProgress, isCancel) {}

    override fun onSuccess(t: T) {

    }
}
